### PR TITLE
Update permissions.yml for bumblebee

### PR DESCRIPTION
**Pull request by jaikev**

### CODE REVIEW

**Comment by jaikev** [comment]
Angel please check this

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.transform
+        - ch.alias.untransform
+groups:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: remove this groups node

**Comment by malonnnn** [comment]
bumblebee is ambiguous, please be more specific on which hero this is

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -1,4 +1,32 @@
 groups:
+  bumblebee:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: bumblebeedc

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -1,4 +1,32 @@
...
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.carryplayer
+        - ch.alias.dismountplayer
+        - ch.alias.transform
+        - ch.alias.untransform
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: what is transform/untransform for?

Comment (jaikev) [inline PermissionsEx/permissions.yml]: Hes an Autobot Optimus Prime got the same thing

**Pull request closed without merging**